concent_api/core/tests/utils.py

```diff
@@ -1080,6 +1082,28 @@ def claim_deposit_false_mock(
     ):  # pylint: disable=unused-argument, no-self-use
         return (None, None)
 
+    def settle_overdue_acceptances_mock(self, requestor_ethereum_address, provider_ethereum_address, acceptances, requestor_public_key):  # pylint: disable=unused-argument, no-self-use
+        requestor_client = Client.objects.get_or_create_full_clean(
+            public_key=requestor_public_key,
+        )
+        requestor_deposit_account = DepositAccount(
+            ethereum_address=requestor_ethereum_address,
+            client=requestor_client,
+        )
+        requestor_deposit_account.full_clean()
+        requestor_deposit_account.save()
+
+        claim_against_requestor = DepositClaim(
+            payee_ethereum_address=provider_ethereum_address,
+            amount=10,
```

Comment:
could the amount be an (optional) param? Or maybe it could come from a const?

Reply:
> could the amount be an (optional) param? Or maybe it could come from a const?

It could come from acceptances list, as it is calculated in bankster, but there is no need for this now.